core/bourbon/helpers/_contrast.scss

```diff
@@ -0,0 +1,31 @@
+@charset "UTF-8";
+
+/// Programatically determines the contrast between two colors.
+///
+/// Note that the alpha channel is ignored.
+///
+/// @link https://www.w3.org/TR/WCAG20/#contrast-ratiodef
```

Comment:
We’ve been consistently using short URL’s for `@link`, so this can be `https://goo.gl/54htLV`.